src/test/java/codeu/service/BBCodeParserTest.java

```diff
@@ -0,0 +1,64 @@
+package codeu.service;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
```

Comment:
Remove

Reply:
Same here, do you mean remove the newline or the imports?

Reply:
same

Reply:
Since you asked for a reference to the guide, section 3.3.3 mentions spacing with imports ([here](https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing))